### DISCUSSION TITLE
VR support

### DISCUSSION THREAD

**Discussion by Neptilo** [pmndrs/postprocessing#423] [Q&A]
Hi,

Is VR supported?

I was using THREE.js's post-processing and tweaked it to make some effects work in VR, especially `OutlinePass`. After the many updates in THREE.js, my tweaks no longer work and I haven't been able to find a fix in the latest revisions. So I am wondering if this repo would provide solutions.

The only mentions of VR I have found in this repo are outdated questions from 2017, and the demos don't have a VR mode.

**Comment by vanruesc** [comment]
Hi, WebXR/VR is currently not supported since the existing passes and effects aren't stereo aware. The current focus lies on improving the render pipeline to support more effects with better performance and flexibility. VR support can be revisited when that's done.
